base/applications/regedit/edit.c

```diff
@@ -1964,16 +1967,22 @@ BOOL ModifyValue(HWND hwnd, HKEY hKey, LPCWSTR valueName, BOOL EditBin)
 
 done:
     if (resourceValueData)
+    {
         HeapFree(GetProcessHeap(), 0, resourceValueData);
-    resourceValueData = NULL;
+        resourceValueData = NULL;
+    }
 
     if (stringValueData)
+    {
         HeapFree(GetProcessHeap(), 0, stringValueData);
-    stringValueData = NULL;
+        stringValueData = NULL;
+    }
 
     if (requirementsValueData)
+    {
         HeapFree(GetProcessHeap(), 0, requirementsValueData);
-    requirementsValueData = NULL;
+        requirementsValueData = NULL;
+    }
```

Comment:
(For this three ifs): Original code was fine, please revert.

Reply:
Well, my point is to use the same pattern here too, instead of setting to `NULL` variables that can already be, which looks useless/odd.